eclass/rpm-extended.eclass

```diff
@@ -0,0 +1,131 @@
+# Copyright 1999-2020 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+# @ECLASS: rpm-extended.eclass
+# @MAINTAINER:
+# Andrew Ammerlaan <andrewammerlaan@riseup.net>
+# @AUTHOR:
+# Andrew Ammerlaan <andrewammerlaan@riseup.net>
+# Extension of the rpm.eclass by Alastair Tse <liquidx@gentoo.org>
+# @SUPPORTED_EAPIS: 6 7
+# @BLURB: An eclass that helps automating the creation of ebuilds for software only distributed through rpm
+# @DESCRIPTION:
+# This extends the rpm eclass to also installs *all* files in the rpm,
+# and extracts post/pre(un)install scripts and runs them.
+#
+# This is mostly useful when creating ebuilds for rpm files in bulk.
+# In cases when it is not doable to write a Gentoo specific src_install
+# and pkg_post/pre/install/rm functions for each ebuild.
+
+# The rpm eclass provides the pkg_unpack function we need
+inherit rpm
+
+case "${EAPI:-0}" in
+	0|1|2|3|4|5)
+		die "Unsupported EAPI=${EAPI:-0} (too old) for ${ECLASS}"
+		;;
+	6|7)
```

Comment:
Do we need 6? We can simplify life by just keeping it 7 yea?

Reply:
Well no we don't really need 6, but from what I understand there are some eclasses that are not compatible with 7 yet (gnome-something if I remember correctly), by keeping 6 in we make sure this eclass is compatible with those eclasses.

Reply:
I'd say, let's remove 6. Saves future work. 
Highly unlikely we are going to be crossing paths with gnome.

Reply:
Alright, 6 is gone :)